docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
+
+### Creating Expectations automatically
+
+As with creating Expectations interactively, you will start with your Data Context.  However, in this case you will be working in a Python environment, so you will need to loading or create your Data Context as an instantiated object.  Next, you will create a Batch Request to specify the data you would like to profile in order to create your Expectation Suite.  Once you have a Batch Request configured you will use it and the name for your Expectation Suite to instantiate a Validator.  The Validator will be used to instantiate a Profiler, and you will then use the `build_suite()` method of the profiler to create your Expectation Suite. 
+
+The Profiler we recommend using for this workflow is the `UserConfigurableProfiler`.  We provide an in depth guide to this in our documentation on [how to create and edit expectations with a profiler](./how_to_create_and_edit_expectations_with_a_profiler.md).
+
+We also provide an in depth guides on [how to create a new Expectation Suite using rule based Profilers](./advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md) and [how to create a new Expectation Suite by profiling from a json schema file](./advanced/how_to_create_a_new_expectation_suite_by_profiling_from_a_jsonschema_file.md) in our advanced how to guides for Expectations.
+
+### Testing your Expectation Suite
+
+Once you have created your Expectation Suite and saved it, you may wish to test it.  The simplest way to do this is to Validate some data against it.  You can do this using a SimpleChekpoint as demonstrated in the [optional step on running validation, saving your suite, and building Data Docs](./how_to_create_and_edit_expectations_with_a_profiler.md#6-optional-running-validation-saving-your-suite-and-building-data-docs) of our [how to create and edit Expectations with a Profiler](./how_to_create_and_edit_expectations_with_a_profiler.md) documentation.  Or you can just move on to [Step 4: Validate Data.](../validation/validate_data_overview.md)
+
+### Editing a saved Expectation Suite
+
+It may be that you have saved an Expectation Suite that you wish to go back to and edit.  The simplest way to do this is to use the CLI.  You can use the command:
+
+```markdown title="Terminal command"
+great_expectations suite edit NAME_OF_YOUR_SUITE_HERE
+```
+
+This will open a Jupyter Notebook that contiains the configurations for each of that Expectation Suite's Expectations in their own cells.  You can edit these cells and then run them to generate Expectations in a new Expectation Suite.  Once your edited version of the Expectations have been created in their own Expectation Suite, you can save that Expectation Suite over the pre-existing one, or save it as a new suite altogether.
```

Comment:
```suggestion
This will open a Jupyter Notebook that contains the configurations for each of that Expectation Suite's Expectations in their own cells.  You can edit these cells and then run them to generate Expectations in a new Expectation Suite.  Once your edited version of the Expectations have been created in their own Expectation Suite, you can save that Expectation Suite over the pre-existing one, or save it as a new suite altogether.
```